ruby_challenges.rb

```diff
@@ -2,7 +2,41 @@
 
 # --------------------1) Create a class called Bike that is initialized with a model, wheels, and a frame size. The default number of wheels is 2. Create a get_info method that returns a sentence with all the data about each bike object.
 
+class Bike
+    attr_accessor :model, :wheels, :frame_size, :speed
+    def initialize (model, wheels, frame_size, speed)
+        @model = model
+        @wheels = 2
```

Comment:
Consider where the defaulting needs to happen and whether you want wheels to ever have more than two wheels.